src/components/icon-button/icon-button.tsx

```diff
@@ -22,7 +25,7 @@ export class IconButton {
      * The icon to display.
      */
     @Prop({ reflect: true })
```

Comment:
Since this can now take a complex type as input, we should perhaps remove the `{ reflect: true }`…

Reply:
sounds great!